media/css/firefox/whatsnew/whatsnew-lite-in.scss

```diff
@@ -63,148 +63,153 @@
 //* -------------------------------------------------------------------------- */
 // Page content
 
-.lite-container {
-    display: flex;
-    justify-content: flex-end;
-    align-items: center;
-    flex-direction: column;
-    margin-top: $spacing-md;
+.lite-body {
+    background-color: #ade7e2;
+    background-image: url('/media/img/firefox/whatsnew/lite/background-in.jpg');
+    background-position: top;
+    color: $color-gray-80;
+    position: relative;
+    @include font-firefox;
+}
 
-    @media #{$mq-lg} {
-        margin-bottom: $spacing-md;
+.lite-content {
+    max-width: 1600px;
+    padding-left: $spacing-lg;
+    padding-right: $spacing-lg;
+    background-repeat: no-repeat;
+    background-position: top right;
+    background-size: 42%;
+    @media #{$mq-md} {
+        background-image: url('/media/img/firefox/whatsnew/lite/photo-in.webp');
+        padding-bottom: $layout-xl;
+    }
+}
+
+.lite-copy {
+    @include text-body-md;
+    @media #{$mq-md} {
+        width: 58%;
     }
 }
 
+
 .lite-heading {
     display: flex;
     align-items: center;
     justify-content: center;
-
-    @media #{$mq-md} {
-        min-width: 500px;
-    }
+    margin-top: $spacing-xl;
+    margin-bottom: $spacing-xl;
 
     h1 {
-        @include font-firefox;
-        @include text-display-lg;
-
         font-weight: normal;
-        text-align: start;
+        font-size: 2rem;
         margin-bottom: 0;
+        text-align: start;
+
+        @media #{$mq-sm} {
+            font-size: 3rem;
```

Comment:
Try to avoid setting arbitrary font sizes if at all possible. Instead I'd suggest trying to use one of the predefined typescales from Protocol: https://protocol.mozilla.org/fundamentals/typography.html